apps/src/tutorialExplorer/filterGroupSortBy.jsx

```diff
@@ -0,0 +1,64 @@
+/* SortBy: The sort order of tutorials.
+ */
+
+import React, {PropTypes} from 'react';
+import FilterGroupContainer from './filterGroupContainer';
+import { TutorialsSortBy } from './util';
+import i18n from './locale';
+
+const styles = {
+  select: {
+    width: '100%',
+    marginTop: 10,
+    height: 26
+  }
+};
+
+const FilterGroupSortBy = React.createClass({
```

Comment:
Please use ES6 classes for new React components.  [`React.createClass` is deprecated](https://reactjs.org/blog/2017/04/07/react-v15.5.0.html#new-deprecation-warnings) as of v15.5.0.

Let me know if you want any guidance on this.  Common gotchas include:
- The `getInitialState` function is no longer supported.
- The `isMounted` function is no longer supported.
- Component methods are not autobound, so you should express callbacks (like `handleChangeSort` here) as arrow functions.